src/gui/lazyfilelist-c-api.h

```diff
@@ -0,0 +1,20 @@
+// temporary C API for the lazy dir view
+#pragma once
+#include <stdint.h>
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+bool LDV_ChangeDir(void *LDV, bool sortByName, const char *path);
+uint32_t LDV_TotalFilesCount(void *LDV);
+uint32_t LDV_VisibleFilesCount(void *LDV);
+bool LDV_MoveUp(void *LDV);
+bool LDV_MoveDown(void *LDV);
+const char *LDV_FileAt(void *LDV, int index, bool *isFile);
+void *LDV_Create(void);
```

Comment:
Also, what about adding `typedef void* LDV_ptr_t`, so random people don't run away when seeing all the `void *` around? 😁

Reply:
The C-API is just a temporary solution to plant the new implementation into the existing code, it will be removed soon, so I'd stick with the void * for now.